turbine/src/retransmit_stage.rs

```diff
@@ -331,16 +331,15 @@ fn retransmit_shred(
         .fetch_add(compute_turbine_peers.as_us(), Ordering::Relaxed);
 
     let mut retransmit_time = Measure::start("retransmit_to");
+    let num_addrs = addrs.len();
     let num_nodes = match cluster_nodes::get_broadcast_protocol(key) {
-        Protocol::QUIC => addrs
-            .iter()
-            .filter_map(|addr| {
-                quic_connection_cache
-                    .get_connection(addr)
-                    .send_data(shred)
-                    .ok()
-            })
-            .count(),
+        Protocol::QUIC => {
+            let shred = Bytes::from(Vec::from(shred));
```

Comment:
I read a bit more about `Bytes` because this usage seemed weird, but it's really neat! As a nit, you could do:

```suggestion
            let shred = Bytes::copy_from_slice(shred);
```

But it does the same thing underneath anyway :shrug:  https://docs.rs/bytes/latest/bytes/struct.Bytes.html#method.copy_from_slice